schemas/sampler_v2_schema.json

```diff
@@ -0,0 +1,85 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "id": "http://www.qiskit.org/schemas/sampler_v2_schema.json",
+    "title": "SamplerV2 input",
+    "description": "The input for an SamplerV2 API call",
+    "version": "1.0.0",
+    "type": "object",
+    "required": ["pubs"],
+    "properties": {
+        "pubs": {
+            "type": "array",
+            "description": "Primitive Unit Blocs of data. Each PUB is of the form (Circuit, Parameters, Shots) where the circuit is required, parameters should be passed only for parametrized circuits, and shots is optional",
+            "items": {
+                "type": "array",
+                "minItems": 1,
+                "prefixItems": [
+                    {"description": "The quantum circuit in QASM string or base64-encoded QPY format. See https://docs.quantum.ibm.com/api/qiskit/qpy for more details on QPY.", "type": "object"},
```

Comment:
I think this type should be an `object` or a `string` (for the qasm string case)

Reply:
Done in `b9f86871958419765806ec4e41c2810d75351fc5`